de.hamstersimulator.objectsfirst.inspector/src/main/java/de/hamstersimulator/objectsfirst/inspector/ui/InputControl.java

```diff
@@ -208,6 +336,15 @@ private void setFromString(final String textValue) {
         }
     }
 
+    /**
+     * Sets the current value
+     * changeFromInputControl is temporarily set to true, to
```

Comment:
Remove the comma.

Reply:
removed with d5ae83d9b6d00633c06a658d67118784234e9597